src/script/calling/CallingRepository.ts

```diff
@@ -153,7 +155,14 @@ export class CallingRepository {
   }
 
   toggleSftCalling(enableSftCalling: boolean) {
-    this.useSftCalling(this.supportsConferenceCalling && enableSftCalling);
+    const urlSetting = UrlUtil.getURLParameter(QUERY_KEY.ENABLE_SFT_CALLING);
+    if (urlSetting) {
```

Comment:
This will trigger if `enable_sft_calling` is set to `'false'`, right?

```suggestion
    if (urlSetting === 'true') {
```

Reply:
Yes, but it should trigger whenever the param is set.